packages/css/src/shorthand-parser/index.ts

```diff
@@ -129,31 +133,85 @@ export const animation = createPropertyParser(
 );
 
 export const font = createPropertyParser(
-  (tokens: any, css: any, value: any) => {
-    if (value.match(/^[0-9.]+deg/)) css.fontStyle += ` ${value}`;
-    else if (value.match(/\//)) {
-      const [fontSize, lineHeight] = value.split("/");
-      css.fontSize = tokens.fontSizes[fontSize] || fontSize;
-      css.lineHeight = tokens.lineHeights[lineHeight] || lineHeight;
-    } else if (
-      value.match(unitMatch) ||
-      value.match(
-        /xx-small|x-small|small|medium|large|x-large|xx-large|xxx-large/
-      ) ||
-      tokens.fontSizes[value]
-    )
-      css.fontSize = tokens.fontSizes[value] || value;
-    else if (value.match(/normal|italic|oblique/)) css.fontStyle = value;
-    else if (value.match(/normal|bold/) || tokens.fontWeights[value])
-      css.fontWeight =
-        value === "normal" ? 400 : tokens.fontWeights[value] || 700;
-    else if (value.match(unitMatch) || tokens.fontSizes[value])
-      css.fontSize = tokens.fontSizes[value] || value;
-    else {
+  (
+    tokens: any,
+    css: any,
+    value: any,
+    index: any,
+    chain: any,
+    chainIndex: any,
+    chains: any
+  ) => {
+    if (chains.shouldParseFontFamily) {
       css.fontFamily = setChainedValue(
         css.fontFamily,
         tokens.fonts[value] || value
       );
+    } else {
+      const lower = value.toLowerCase();
+
+      switch (true) {
+        case fontStyleMatch.test(lower):
+          css.fontStyle += ` ${value}`;
+
+          break;
+
+        case fontWeightMatch.test(lower):
+          css.fontWeight = tokens.fontWeights[value] || value;
+
+          break;
+
+        case fontSizeMatch.test(lower):
+          chains.shouldParseFontFamily = true;
```

Comment:
For this, I may deserve the bell of shame. 🔔😰 This is me, a new contributor, desperate to avoid modifying `createPropertyParser`, resorting to adding named keys to the `chains` array.   This is done to ensure that all property parts after the font size are interpreted as font families. ([see usage](https://github.com/modulz/stitches/pull/152/files#diff-0ec06497f9094ba10b71f75d395050c6R145))